site/pages/cookies.tsx

```diff
@@ -0,0 +1,144 @@
+import { parseCookies } from "nookies";
+import { ReactElement } from "react";
+import CsrfForm from "../components/CsrfForm";
+import { TwoThirdsLayout } from "../components/layout/Layout";
+import { COOKIES_POLICY_COOKIE } from "../constants";
+import { NextPageContextWithSession } from "../interfaces";
+import { getCsrfToken } from "../utils";
+
+const title = "Cookies - Create Transport Disruption Data Service";
+const description = "Cookies Preferences page of the Create Transport Disruption Data Service";
+
+export interface CookiePreferencesProps {
+    settingsSaved: boolean;
+    trackingDefaultValue: "on" | "off";
+    csrfToken: string;
+}
+
+const Cookies = ({ settingsSaved, trackingDefaultValue, csrfToken }: CookiePreferencesProps): ReactElement => (
+    <TwoThirdsLayout title={title} description={description} hideCookieBanner>
+        {settingsSaved ? (
+            <div className="information_box__success block p-[20px] mb-[50px] border-4 border-govGreen">
```

Comment:
We can use the built-in tailwind sizes here rather than override with [20px] and [50px], `p-5 mb-12.5` should work

Reply:
done